dts/bindings/rtc/microcrystal,rv3028.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2024 ANITRA system s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Micro Crystal RV3028 RTC
+
+compatible: "microcrystal,rv3028"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+  clkout-frequency:
+    type: int
+    default: 32768
+    enum:
+      - 32768
+      - 8192
+      - 1024
+      - 64
+      - 32
+      - 1
+      - 0
+    description: |
+      Frequency of the CLKOUT signal in Hertz (Hz). Default is 32768 Hz which corresponds to the
+      default value on delivery. Set to 0 to disable the CLKOUT signal (CLKOUT pin is LOW).
+
+  backup-switchover:
+    type: string
+    required: true
+    enum:
+      - disabled
+      - direct
+      - level
+    description: |
+      Automatic backup switchover function selection:
+        - disabled: The switchover function is disabled - only one power supply available (VDD)
+        - direct: Direct Switching Mode (DSM): when VDD < VBACKUP, switchover occurs from VDD to
+        VBACKUP without requiring VDD to drop below VTH:LSM (2.0 V)
+        - level: Level Switching Mode (LSM): when VDD < VTH:LSM (2.0 V) AND VBACKUP > VTH:LSM,
+        switchover occurs from VDD to VBACKUP
+
+  aux-voltage-chargeable:
+    type: int
```

Comment:
should this be a boolean?

Reply:
it could definitely be a boolean, but I thought the intent was to have DT bindings similar to Linux where this property is an integer with enum values

Reply:
It would be helpful to have a link or something then. It looks like this property is generic to RTC in Linux.

Reply:
here is the generic RTC binding in the Linus's tree: [https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/rtc/rtc.yaml](https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/rtc/rtc.yaml)

Reply:
Here is the existing binding in the Linus's tree:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/rtc/microcrystal%2Crv3028.yaml

And the generic RTC binding:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/rtc/rtc.yaml

Reply:
@decsny should I change the datatype or leave it like the existing Linux bindings?

Reply:
ok, sounds good :)

Reply:
@bjarki-trackunit I’ve ended up removing the property `aux-voltage-chargeable` because I felt it was redundant:
https://github.com/zephyrproject-rtos/zephyr/blob/5f8769677117be5a5262b7903c05b261e4c56e9d/dts/bindings/rtc/microcrystal%2Crv3028.yaml#L40-L54